eslaikouskaya/hw3/forms.html

```diff
@@ -0,0 +1,162 @@
+                        <input type="checkbox" name="system" checked>Оставаться в системе
+                    </label><br>
+                    <input type="submit" value="Войти"><br>
+                    <em>*-обязательное поле</em>
+                </fieldset>
+            </form></td> 
+             
+            <td><h3>Опрос (голосование):</h3>
+            <form method="get" action="http://sc.it-academy.by/fe/form.php">
+                <fieldset>
+                    <legend>Как вам?</legend>
+                    <label>
+                        <input type="radio" name="select" value="normal" checked>
+                        ничего
+                    </label><br>
+                    <input type="hidden" name="data_voite" value="2013.12.31">
+                    <label>
+                        <input type="radio" name="select" value="ok">
+                        неплохо
+                    </label><br>   
+                    <label>
+                        <input type="radio" name="select" value="good">
+                        хорошо
+                    </label><br>
+                    <label>
+                        <input type="radio" name="select" value="super">
+                        супер
+                    </label><br> 
+                    <input type="submit" value="Отпрвить">       
+                 </fieldset>
+            </form><td>
+            </tr>
+            
+            <tr>
+            <td><h3>Отзыв:</h3>
+            <form method="post" action="http://sc.it-academy.by/fe/form.php">
+                <fieldset>
+                    <legend>Оставьте Ваш отзыв</legend>
+                    <input type="text" ID="login" name="login" autofocus required>
+                    <label for="login">*Имя</label><br>
+                    <input type="email" name="email" placeholder="это обязательное поле" required>
+                    <label for="pass">*email</label><br>
+                    <textarea name="text_area" lang="ru">
+                    Форма отправки
+                    </textarea>
+                    <input type="submit" value="Отправить"><br>
+                    <em>*-обязательное поле</em>
+                </fieldset>
+            </form></td> 
+            
+            <td><h3>Выпадающий и мульти списки:<h3>
+            <form method="get" action="http://sc.it-academy.by/fe/form.php">
+                <fieldset>
+                    <legend>Города РБ</legend>
+                    <select name="list" id="list">
+                        <option value="1">Минск</option>
+                        <option value="2">Брест</option>
+                        <option value="3">Могилев</option>
+                        <option value="4">Гродно</option>
+                        <option value="5">Борисов</option>
+                        <option value="0" selected>Город</option>
+                    </select>
+                    <select name="list[]" size="3" multiple id="list">
```

Comment:
value on name is duplicating with line 110, all names inside one form should be unique except radio)